server/views/pages/overview/partials/timelineTab/timelineTabContents.njk

```diff
@@ -3,25 +3,26 @@
 
     <div class="moj-timeline">
 
-      <!-- Timeline item -->
+      {% for event in timeline.events %}
+        <div class="moj-timeline__item">
+          <div class="moj-timeline__header">
+            <h2 class="moj-timeline__title">{{ event.eventType | formatTimelineEvent }}</h2>
+            <p class="moj-timeline__byline govuk-!-display-none-print">by {{ event.actionedByDisplayName }}, {{ event.prisonId }}</p>
```

Comment:
For now we're using `prisonId` as we don't have a method for retrieving the prison name 

Note: Hidden on Timeline as per requirements of the Print ticket